api_tests/lib/bitcoin.js

```diff
@@ -47,38 +48,30 @@ module.exports.btc_activate_segwit = async function() {
     return create_bitcoin_rpc_client().generate(432);
 };
 
-module.exports.btc_import_address = async function(address) {
-    return create_bitcoin_rpc_client().importAddress(address);
-};
+async function get_satoshi_transferred_to(tx_id, address) {
+    let satoshi = 0;
+    let tx = await _bitcoin_rpc_client.getRawTransaction(tx_id, 1);
+    let vout = tx.vout[0];
 
-async function btc_balance(address) {
-    let btc_balance = await _bitcoin_rpc_client.getReceivedByAddress(address);
-    return parseFloat(btc_balance) * 100000000;
-}
+    if (
+        vout.scriptPubKey.addresses.length === 1 &&
+        vout.scriptPubKey.addresses[0] === address
+    ) {
+        satoshi = sb.toSatoshi(vout.value);
+    }
 
-module.exports.btc_balance = async function(address) {
-    return btc_balance(address);
-};
+    return satoshi;
+}
 
-module.exports.log_btc_balance = async function(
-    when,
-    player,
-    address,
-    address_type
-) {
-    global.harness.logger.info(
-        "%s the swap, %s has %s satoshis at the %s address %s",
-        when,
-        player,
-        await btc_balance(address),
-        address_type,
-        address
-    );
+module.exports.get_satoshi_transferred_to = async function(tx_id, address) {
```

Comment:
You should be able to just assign the fn here:

```suggestion
module.exports.get_satoshi_transferred_to = get_satoshi_transferred_to;
```